coordinator/src/node.rs

```diff
@@ -672,6 +673,16 @@ impl Node {
                     "Opening positions is disabled"
                 );
 
+                ensure!(
+                    !self
+                        .inner
+                        .list_dlc_channels()?
+                        .iter()
+                        .filter(|c| c.get_counter_party_id() == trader_peer_id)
+                        .any(|c| matches!(c, Channel::Offered(_) | Channel::Accepted(_))),
+                    "Previous DLC Channel offer still pending."
```

Comment:
How do we recover from this? 
Does an `Offer` time out?

Reply:
The app needs to reject the offer.

Reply:
Thanks for the explanation.